### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by rollerchloe**
## Summary

This readme is updated so that it can be retrieved as a remote file in the documentation website.

## Changes

- Proofread the content 
- Update broken links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,60 +181,71 @@ const action = {
...
+- `document`: (mandatory) Document data as returned by the `getData` method from [@govtechsg/open-attestation](https://github.com/Open-Attestation/open-attestation)
+- `rawDocument`: (optional) OpenAttestation document
+- `handleObfuscation`: (mandatory) A function to call that will handle obfuscation in the document. The value provided must follow the path property as handled by [lodash#get](https://lodash.com/docs/4.17.15#get).
 
 ## Development
 
-- `npm run storybook`: to start storybook, create stories and visualize the different component
+Run the following commands for different development tasks:
+
+- `npm run storybook`: to start a storybook, create stories, and visualize the different component
```
Comment (yapyuyou) [inline README.md:237]:
```suggestion
- `npm run storybook`: to start a storybook, create stories, and visualize different components
```

File: README.md [hunk 2]
```diff
@@ -146,60 +181,71 @@ const action = {
...
 - `npm run test`: to run tests
 - `npm run lint`: to run lint
-- `npm run example:application`: to run the example build with the library to develop an hosting application. Don't forget to update the example if you update this library.
-- `npm run example:renderer`: to run the example build with the library to develop a decentralized renderer. Don't forget to update the example if you update this library.
+- `npm run example:application`: to run the example build with the library to develop an hosting application.
```
Comment (yapyuyou) [inline README.md:240]:
```suggestion
- `npm run example:application`: to run an example application built with this library. 
```

File: README.md [hunk 3]
```diff
@@ -146,60 +181,71 @@ const action = {
...
+
+- `npm run storybook`: to start a storybook, create stories, and visualize the different component
 - `npm run test`: to run tests
 - `npm run lint`: to run lint
-- `npm run example:application`: to run the example build with the library to develop an hosting application. Don't forget to update the example if you update this library.
-- `npm run example:renderer`: to run the example build with the library to develop a decentralized renderer. Don't forget to update the example if you update this library.
+- `npm run example:application`: to run the example build with the library to develop an hosting application. 
+
+  Be sure to update the example if you update this library.
+- `npm run example:renderer`: to run the example build with the library to develop a decentralized renderer.
```
Comment (yapyuyou) [inline README.md:243]:
```suggestion
- `npm run example:renderer`: to run an example decentralized renderer built with this library. 
```

**Review by yapyuyou — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -146,60 +181,71 @@ const action = {
...
 
 ### FramedDocumentRenderer
 
-This component will establish a connection with a host embedding the application within an iframe. Properties are:
+This component will establish a connection with a host embedding the application within an iframe. Its properties include:
+
+- `templateRegistry`: The configuration of the templates handled by the decentralized renderer. `templateRegistry` is an object where each key holds an array of `Template Configuration`. 
+
+  One `Template Configuration` consists of:
+  - `id`: A unique identifier withing the current array of the template
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:206]: sorry for typo - fixed

File: README.md [hunk 5]
```diff
@@ -146,60 +181,71 @@ const action = {
...
+  - `label`: A string to represent what the template is
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:207]: added

File: README.md (hunk 1 above)
Comment (rollerchloe) [reply README.md:238]: updated

File: README.md (hunk 2 above)
Comment (rollerchloe) [reply README.md:241]: updated

File: README.md (hunk 3 above)
Comment (rollerchloe) [reply README.md:244]: updated

**Review by yapyuyou — APPROVED** [review]

**Pull request merged**